### ISSUE TITLE
training with custom data

### ISSUE THREAD

**Issue by AhmadHakami** [MelosY/CAM#1]
@MelosY thank you for your excellent work

i'm wondering now if it is possible to train the model on my own data, additionally i would be grateful if you could explain the data specifications and format in more detail because this would make it easier and clearer for everyone to use your model and collaborate on this repository to enhance it

**Comment by MelosY** [comment]
Yes, it can be trained on your own data. We organize the data into lmdb type, which is familiar for text recognition.  The specific conversion method can be referred to https://github.com/ayumiymk/aster.pytorch/blob/master/lib/tools/.
